openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py

```diff
@@ -1,193 +1,282 @@
+"""
+Content Provider:       Rawpixel
+
+ETL Process:            Use the API to identify all CC-licensed images.
+
+Output:                 TSV file containing the image meta-data.
+
+Notes:                  Rawpixel has given us beta access to their API.
```

Comment:
Would it be clearer to say 'Rawpixel has given _Openverse_ beta access'? It might not be clear who `we` are in the docs.

Reply:
Oh that's a great point, I'll update this!